src/index.js

```diff
@@ -6,15 +6,25 @@ import { createStore, applyMiddleware, compose } from 'redux'
 import { rootReducer } from './redux/rootReducer'
 import { Provider } from 'react-redux'
 import thunk from 'redux-thunk'
+import { loadState, saveState } from './localStorage'
+
+const persistedState = loadState()
 
 const store = createStore(
   rootReducer,
+  persistedState,
   compose(
     applyMiddleware(thunk),
     window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
   )
 )
 
+store.subscribe(() => {
+  saveState({
+    userInfo: store.getState().userInfo,
+  })
+})
+
```

Comment:
move create a store and all logic with the store to redux folder

Reply:
fixed